lib/puppet/indirector/rest.rb

```diff
@@ -149,7 +149,7 @@ def http_request(method, request, *args)
     conn.send(method, *args)
   end
 
-  def find(request)
+  def find(request, &block)
```

Comment:
I was thinking of adding this to all of the `REST` methods for consistency, e.g. search, delete, etc